docs/CUSTOMIZATION.md

```diff
@@ -0,0 +1,140 @@
+
+By default, with no tile size set, the view will scale to fill as much space as it can.
+The functionality is similar to ImageView with `adjustViewBounds` set to true.
+
+If a tileSize is set, that will override the `layout_width` and `layout_height` set.
+
+The view is 7 tiles wide and 8 tiles high (with the top bar visible).
+
+
+## Date Selection
+
+We support three modes of selection: single, multiple, or none. The default is single selection.
+The mode can be changed by calling `setSelectionMode()` and passing the appropriate constant (`SELECTION_MODE_NONE`, `SELECTION_MODE_SINGLE`, or `SELECTION_MODE_MULTIPLE`).
+If you change to single selection, all selected days except the last selected will be cleared.
+If you change to none, all selected days will be cleared.
+
+You can set an `OnDateSelectedListener` to listen for selections, make sure to take into account multiple calls for the same date and state.
+You can manually select or deselect dates by calling `setDateSelected()`.
+There is also `setSelectedDate()` will will clear the current selection(s) and select the provided date.
+
+There are also: `clearSelection()`, `getSelectedDates()`, and `getSelectionMode()`; which should be self-explanitory.
+
+
+## Showing Other Dates
+
+By default, only days of one month, in the min-max range, are shown.
+You can customize this by setting `mcv_showOtherDates` in xml, or by calling `setShowOtherDates()`.
+
+In xml, there are flags for `other_months`, `out_of_range`, and `decorated_disabled`, and also `none`, `all`, or `defaults` for convenience.
+
+There are similar constants on `MaterialCalendarView` such as `MaterialCalendarView.SHOW_DECORATED_DISABLED`.
+
+
+## Selection Color
+
+The default color of the calendar selector is the one set referenced by `?android:attr/colorAccent` on 5.0+ or `?attr/colorAccent` from the AppCompat library (black as a last resort).
+You can provide a custom color by setting `mcv_selectionColor` in xml, or by calling `setSelectionColor()`.
+
+If you want more control than just color, you can use the [decorator api](DECORATORS.md) to set a [custom selector](CUSTOM_SELECTORS.md).
+
+
+## First Day Of The Week
+
+The default first day of the week is Sunday. You can set a custom day of the week by setting `mcv_firstDayOfWeek` in xml, or by calling `setFirstDayOfWeek()`.
+The xml attribute is an enum of `sunday` through `saturday` and `setFirstDayOfWeek()` accepts values from `java.util.Calendar` such as `Calendar.MONDAY`.
+
+
+## Date Ranges
+
+By default, the calendar displays months for 200 years before today and 200 years after.
+You can specify different minimum and maximum dates by calling `setMinimumDate(CalendarDay)` and `setMaximumDate(CalendarDay)`.
+Passing `null` will reset back to the default 200 years.
+There are also convience methods that accept a `Calendar` or a `Date` object, which convert them to a `CalendarDay` using the relevant `CalendarDay.from()` factory method.
```

Comment:
Typo: "convience" should be "convenience"
Re-word to "There are also convenience methods that accept a `Calendar` or `Date` object and convert it to a `CalendarDay` using the relevant `CalendarDay.from()` factory method."